docs/api-reference/webgl/texture-3d.md

```diff
@@ -1,15 +1,13 @@
 # Texture3D (WebGL2)
 
-A `Texture3D` holds a number of textures of the same size and format. The entire array can be passed to the shader which uses an extra texture coordinate to sample from it. A core feature of `Texture3D` is that the entire stack of images can passed as a single uniform to and accessed in a GLSL shader, and sampled using 3D coordinates.
```

Comment:
Why cut this? These blurbs were added to all luma.gl doc pages to help new users understand what these webgl objects are about. 

Not that many people know how WebGL and especially WebGL2 classes work.

This could be worth discussing before cutting the rest of the corresponding docs

Reply:
Those docs were for texture arrays. They don't apply to 3D textures.

Reply:
> Those docs were for texture arrays. They don't apply to 3D textures.

Then correct them instead of cut?

When I read the API specs, 3d textures and texture2darrays seemed very similar in that they both basically are arrays of textures and the various textures are accessed by depth, so I felt that the description here was essentially correct.

My initial challenge was to actually to understand what was different between the two, therefore, it could be helpful to users to include more insightful information about differences. Googling [here](https://stackoverflow.com/questions/35818027/understanding-the-difference-between-a-2d-texture-array-and-a-3d-texture) these differences are listed:

* GenerateMipmap will blend cross layers with 3D textures but not 2D arrays
* the Z coordinate passed to texture in GLSL is 0 to 1 with 3D textures but an 0 to N (depth) in 2D arrays.
* If filtering is not NEAREST 3D will blend across layers, 2D array will not.
* In a 3D texture, the width, height, and depth all decrease at lower mipmap sizes. In a 2D array texture, every mipmap level has the same number of array layers; only width and height decrease.

I think it would be neat to add some version of that list to the docs...